lib/index.ts

```diff
@@ -292,30 +258,8 @@ export default class Backburner {
   public schedule(queueName: string, method: Function);
   public schedule<T, U extends keyof T>(queueName: string, target: T, method: U, ...args);
   public schedule(queueName: string, target: any | null, method: any | Function, ...args);
-  public schedule(queueName: string) {
-    let length = arguments.length;
-    let method;
-    let target;
-    let args;
-
-    if (length === 2) {
-      method = arguments[1];
-      target = null;
-    } else {
-      target = arguments[1];
-      method = arguments[2];
-
-      if (isString(method)) {
-        method = <Function> target[method];
-      }
-
-      if (length > 3) {
-        args = new Array(length - 3);
-        for (let i = 3; i < length; i++) {
-          args[i - 3] = arguments[i];
-        }
-      }
-    }
+  public schedule() {
```

Comment:
Since `queueName` is required, I believe you can restructure this to use the same logic as `parseArgs`:

```js
public schedule(queueName, ..._args) {
  let [target, method, args] = parseArgs(_args);
}
```